### PR TITLE
Frr shop test

### PR DESCRIPTION
**Pull request by rfrancel**
@oterral here is the branch for the tooltip fir the shop layers, you can start to work on the shop button

### CODE REVIEW

File: chsdi/templates/htmlpopup/skitouren_metadata_test.mako [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+<%def name="table_body(c, lang)">
+
+<%
+   lang = lang if lang in ('fr','it','en') else 'de'
+%>
+
+<%
+   fid = str(c['featureId'])
+   image = "https://dav0.bgdi.admin.ch/swisstopoproducts/110/" + fid + ".jpg"
```
Comment (loicgasser) [inline chsdi/templates/htmlpopup/skitouren_metadata_test.mako]: We have a variable for that in the settings, namely `webdav_host`. Use `request.registry.settings` to access it.

File: chsdi/templates/htmlpopup/skitouren_metadata_test.mako [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+
+<%
+   lang = lang if lang in ('fr','it','en') else 'de'
+%>
+
+<%
+   fid = str(c['featureId'])
```
Comment (loicgasser) [inline chsdi/templates/htmlpopup/skitouren_metadata_test.mako]: it's already a string according to your model

File: chsdi/templates/htmlpopup/skitouren_metadata_test.mako [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+   image = "https://dav0.bgdi.admin.ch/swisstopoproducts/110/" + fid + ".jpg"
+%>
+
+
+    <tr><td class="cell-left">${_('kartenblatt')}</td>         <td>${c['featureId']} ${c['attributes']['name']}<td><td rowspan="6"><img
+    src="${image}"/></{{td></tr>
+    <tr><td class="cell-left">${_('datenstand')}</td>          <td>${c['attributes']['legendecms2007']}<td></tr>
+    <tr height="100"><td class="cell-left">${_('preis')}</td>               <td>${c['attributes']['preis']}</td></tr>
```
Comment (loicgasser) [inline chsdi/templates/htmlpopup/skitouren_metadata_test.mako]: we have a bunch of css classes you could use/edit in:
https://github.com/geoadmin/mf-chsdi3/blob/master/chsdi/static/less/extended.less

File: chsdi/locale/de/LC_MESSAGES/chsdi.po [hunk 4]
```diff
@@ -4517,6 +4520,9 @@ msgstr "Datum"
 msgid "date_time"
 msgstr "Datum/Zeit"
 
+msgid "datenstand"
+msgstr "Datenstand"
```
Comment (loicgasser) [inline chsdi/locale/de/LC_MESSAGES/chsdi.po]: we probably already have this translation, please check it out first before adding new ones.

**Comment by AFoletti** [comment]
Is this PR still needed? If not, please close and delete branch. Thanks!

**Pull request closed without merging**